src/extension.ts

```diff
@@ -82,7 +83,11 @@ export async function activate(ctx: vscode.ExtensionContext) {
   controller.createRunProfile("Meson debug test", vscode.TestRunProfileKind.Debug, (request, token) => testDebugHandler(controller, request, token), true)
   controller.createRunProfile("Meson run test", vscode.TestRunProfileKind.Run, (request, token) => testRunHandler(controller, request, token), true)
 
-  let changeHandler = async () => { explorer.refresh(); await rebuildTests(controller); };
+  let changeHandler = async () => {
+    explorer.refresh();
+    await rebuildTests(controller);
+    await genEnvFile(buildDir);
```

Comment:
Looking at the code, this doesn't actually return anything, but rebuilds the env file.

Can you name the function accordingly?

Reply:
I think you misread, it starts with *gen* as in genarate, not *get*.

Reply:
Ahh, right.